HW2/P2/P2.py

```diff
@@ -36,16 +37,22 @@
     assert counts.sum() == total, "Wrong total after move_data_fine_grained"
     print("Fine grained uncorrelated: {} seconds".format(t.interval))
 
+
     ########################################
     # You should explore different values for the number of locks in the medium
     # grained locking
     ########################################
-    N = 10
-    counts[:] = orig_counts
-    with Timer() as t:
-        move_data_medium_grained(counts, src, dest, 100, N)
-    assert counts.sum() == total, "Wrong total after move_data_medium_grained"
-    print("Medium grained uncorrelated: {} seconds".format(t.interval))
+    #we created a new array to put our performance result in
+    uncorrelated = []
+    N = [5,10,20,30,40,50,100]
```

Comment:
You should consider seeing what happens when N is even greater (in the 1000s, for instance)